Feat: Toolbox - URL query parameters connectURL and db (#1547)
neo4j/graphql: packages/graphql-toolbox/README.md

yarn start
```

## GraphQL Toolbox URL parameters

> The GraphQL Toolbox supports some URL parameters defined in the query component, see below. These URL parameters are optional and can be provided at convenience.
> The query component is preceded by a question mark (?) and contains a query string that is a sequence of key-value pairs separated by an ampersand (&).

Reviewer: ```suggestion
The GraphQL Toolbox supports some URL parameters defined in the query string, see below. These URL parameters are optional and can be provided for convenience.
The query string is preceded by a question mark (?) and contains key-value pairs separated by an ampersand (&).
```

I think with this change the language becomes more standardised, so more easily understandable.

| URL parameter | Description                                                                                                                             | Example                     |
| ------------- | --------------------------------------------------------------------------------------------------------------------------------------- | --------------------------- |
| connectURL    | Form: scheme://username@bolt-url:port; Used at login, pre-populates the Username and Connection URI input field with the provided value | bolt://admin@localhost:7687 |
| db            | Only applicable for multi-database supported Neo4j DBMSs. The provided value is used as the selected database name                      | neo4j                       |

> An example with the URL parameters `connectURL` and `db`: `http://localhost:4242?connectURL=bolt%2Bs://testuser@xxxx.databases.neo4j.io&db=mydatabase`

Reviewer 2: In the example there is the Query parameter `db` but then in the login form there is not used, in my opinion this could lead to ambiguity/bugs.

Reviewer 2: Tbf it's more related to the UX aspect of the project rather than an engineering one, what i'm trying to say is that the feedback to the user is weak, also because there is no validation over the DB name, if the DB name is wrong then another one is used (without any feedback), again, nothing wrong from the engineering point of view :)

> Note: The plus symbol (+) in the connectURL (here in `bolt+s`) needs to be URL encoded with `%2B`.

Reviewer: ```suggestion
Note: The plus symbol (+) in the connectURL needs to be URL encoded to become `%2B`. For example `bolt+s` becomes `bolt%2Bs`.
```
I think this may improve clarity.
